### PR TITLE
Show error summary when installation fails because of insufficient available disk space

### PR DESCRIPTION
**Pull request by Pythyu**
Grep specific part of the system's package manager error output to display them at the end to increase visibility when an installation fails because of insufficient available space.

### CODE REVIEW

File: install_script.sh.template
Comment (chouquette) [inline install_script.sh.template:776]: Is displaying the entire log expected or is this a left over debug?

**Review by chouquette — APPROVED** [review]
LGTM, modulo the question about `cat /tmp/ddog_install_error_msg`

File: install_script.sh.template
Comment (Pythyu) [reply install_script.sh.template:776]: I kinda wanted to use something similar to `tee` to store stderr into a file and still show it on stderr this way it doesn't remove the error from the live logs. I didn't find a great alternative since `tee` only works for stdout and wasn't sure if redirecting stderr into stdout would be a great idea. At the moment I'm simply storing stderr and printing it back to avoid any information loss. But still not great because it messes up the order in the log if there's an error. If you have any suggestion I'm open to it 🤔

Comment (chouquette) [reply install_script.sh.template:776]: Forcing errors onto `stdout` might be counter intuitive indeed, although currently we don't display anything and the errors are displayed on `stdout`, so as far as which FD we use, I guess that's fine
My worry was mostly about adding the entire log file to the terminal, since that wasn't the case before, and it might be a bit too verbose, but I'll let you decide as long as the change is intentional :)

File: install_script.sh.template
Comment (chouetz) [inline install_script.sh.template:896]:
```suggestion
    ERR_SUMMARY=$(grep -C1 "No space left on device" /tmp/ddog_install_error_msg || true)
```
I suppose it's equivalent?

File: install_script.sh.template
Comment (chouetz) [inline install_script.sh.template:776]: Shouldn't we remove these `cat`. [I was told we prefer dogs (sorry for this one)]
The only display is the one in the final error message, correct?

**Review by chouetz — COMMENTED** [review]
Do we do `cat` in addition to the final error message in case of errors?

**Review by chouetz — APPROVED** [review]

**Pull request merged**
